extensions/azure-functions-http/runtime/src/main/java/io/quarkus/azure/functions/resteasy/runtime/BaseFunction.java

```diff
@@ -56,7 +56,7 @@
     }
 
     protected HttpResponseMessage dispatch(HttpRequestMessage<Optional<byte[]>> request) {
-        VirtualClientConnection connection = VirtualClientConnection.connect(VertxWebRecorder.VIRTUAL_HTTP);
+        VirtualClientConnection connection = VirtualClientConnection.connect(VertxWebCommonRecorder.VIRTUAL_HTTP);
```

Comment:
I think we should use the Vert.x common artifacts in Azure functions but I wanted to check with @patriot1burke and @stuartwdouglas before.

Reply:
From what I see yes. You don't need the user-facing features (event bus integration, RX Java 2, Axle, @Route)

Reply:
The azure functions extension is very small.  The @BuildStep just requests virtual http.  Runtime only needs a constant from VertxWebRecorder.

Reply:
@patriot1burke OK, I'll move it to use the new low-level extensions in the new PR.